File: 1° Período - Introdução a Programação 1/Exercicios em Python/Lista 1 - Python/q2.py
#Faça um Algoritmo que receba um salário e atribua em cima do salário uma
#gratificação de 5% e depois atribua o imposto de 8% em cima do valor atual. 
salario=float(input("Insira o salario base: "))
salario_receber=salario+(salario*(0.05-0.08))
print("Gratificação de 5% e imposto de 8%:")
print("Salario a receber: ", salario_receber) 

File: 1° Período - Introdução a Programação 1/Exercicios em C/Lista 1 - C/q2.c
// Faça um Algoritmo que receba um salário e atribua em cima do salário uma gratificação de 5% e depois atribua o imposto de 8% em cima do valor atual. 
#include<stdio.h>
int main(){
	float salario, receber, gratificacao, imposto;
	printf("SALARIO: ");
	scanf("%f", &salario);
	receber=salario+(salario*(0.05-0.08));
	printf("SALARIO A RECEBER: %f\n", receber);
	system("pause");
}
